dsl/pace/dsl/stencil.py

```diff
@@ -518,15 +520,11 @@ def closure_resolver(self, constant_args, given_args, parent_closure=None):
 
 def _convert_quantities_to_storage(args, kwargs):
     for i, arg in enumerate(args):
-        try:
-            args[i] = arg.storage
-        except AttributeError:
-            pass
+        if isinstance(arg, pace.util.Quantity):
+            args[i] = arg.data
     for name, arg in kwargs.items():
-        try:
-            kwargs[name] = arg.storage
-        except AttributeError:
-            pass
+        if isinstance(arg, pace.util.Quantity):
+            kwargs[name] = arg.data
```

Comment:
Please use `try: except` as before instead of `isinstance`

Reply:
In this case that doesn't work because both Quantity and ndarrays have a data attribute

Reply:
We could rename the data attribute for now to disambiguate, and/or continue down the path of making Quantity an ndarray instead of holding one.

Reply:
Given where this code is, it's performance critical to use a try:except block here over isinstance. Can you add a do-nothing line that just attempts to access an attribute we know only exists on Quantity, and add a `# code comment` explaining why it's there?

Reply:
Ok I added that.